### ISSUE TITLE
using the dataframe HTML styler changes representation of NaNs

### ISSUE THREAD

**Issue by jrenner** [pydata/pandas#12710]
The regular output of a dataframe with NaN values displays them as "NaN",

when using the HTML styler, they are displayed as "nan"

I'm not sure if this breaks anything but it is an inconsistency.

**Comment by jreback** [comment]
I don't think there is currently any way to actually specify how these display, and they are default rendered. You could do:

``.format(lambda x: 'NaN' if pd.isnull(x) else x)`` I think

cc @TomAugspurger

**Comment by TomAugspurger** [comment]
Yeah, the default should changed. For `.style` it's defined [here](https://github.com/pydata/pandas/blob/247fe0718c3b09a2de4d1af834cff8efb9f8edcc/pandas/core/style.py#L155) but should probably just be reusing pieces from https://github.com/pydata/pandas/blob/master/pandas/core/format.py
